presto-docs/src/main/sphinx/release/release-0.234.rst

```diff
@@ -0,0 +1,49 @@
+=============
+Release 0.234
+=============
+
+.. warning::
+
+   There is a regression in the scheduler that has not been fixed in this release. It has been temporarily mitigated by setting
+   the default value of the configuration property ``use-legacy-scheduler`` to ``true``. Please keep the legacy scheduler enabled
+   until the issue is fully resolved.
+
+General Changes
+_______________
+* Fix query failure for cross-joining bucketed tables with ``COALESCE`` when the bucket counts are compatible but mismatched.
+* Fix query failure for window functions with ``IGNORE NULLS``.
```

Comment:
I thought this was a wrong results issue, rather than a query failure. @kaikalur

Reply:
You're right. I double checked the original PR.